appservice/src/intTest/groovy/nu/yona/server/AbstractAppServiceIntegrationTest.groovy

```diff
@@ -295,6 +295,17 @@ abstract class AbstractAppServiceIntegrationTest extends Specification
 		dateTime.get(ChronoField.MINUTE_OF_DAY)/15
 	}
 
+	def setMandatoryGoalsToNow(User user)
+	{
+		def mandatoryGoals = user.goals.find { it.activityCategoryUrl == GAMBLING_ACT_CAT_URL }
```

Comment:
The ``find`` operation returns just one object, so the method can be this simple:
```
def goal = user.goals.find { it.activityCategoryUrl == GAMBLING_ACT_CAT_URL }
goal.creationTime = YonaServer.now
def response = appService.updateGoal(user, goal.url, goal)
assert response.status == 200
```